index.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Microverse Developers Conference</title>
+    <link rel="shortcut icon" href="images/logo.png" type="image/x-icon">
+    <link rel="stylesheet" type="text/css" media="screen" href="main.css">
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.0/css/all.min.css" rel="stylesheet">
+</head>
+
+<body>
+    <header class="header">
+        <nav class="top-nav">
+            <ul class="top-ul">
+                <li class="top-list"><i class="fab fa-facebook-f"></i></li>
+                <li class="top-list"><i class="fab fa-twitter"></i></li>
+                <li class="top-list">English</li>
+                <li class="top-list">My Page</li>
+                <li class="top-list logout">Logout</li>
+            </ul>
+        </nav>
+        <div class="container">
+            <div class="nav-item3">
+                <span class="hamburger-span"> &#9776; </span>
+            </div>
+            <nav class="web-nav">
+                <nav class="first-header">
+                    <h1>DEVELOPERS<span class="first-span">WEEK</span></h1>
+                </nav>
+                <ul class="web-nav-ul">
+                    <li class="web-nav-list">
+                        <a href="about.html" class="web-nav-link">About</a>
+                    </li>
+                    <li class="web-nav-list">
+                        <a href="#program" class="web-nav-link">Conference</a>
+                    </li>
+                    <li class="web-nav-list">
+                        <a href="#devs" class="web-nav-link">Speakers</a>
+                    </li>
+                    <li class="web-nav-list">
+                        <a href="#partner" class="web-nav-link">Sponsors</a>
+                    </li>
+                    <li class="web-nav-list">
+                        <a href="#" class="web-nav-link active">CN Campaign</a>
+                    </li>
+                </ul>
+            </nav>
+        </div>
+        <div id="myNav" class="overlay">
+            <a class="closebtn"> &times; </a>
+            <div class="overlay-content">
+                <a href="about.html" class="web-nav-link">About</a>
+                <a href="#program" class="web-nav-link">Program</a>
+                <a href="#devs" class="web-nav-link">Developers</a>
+                <a href="#partner" class="web-nav-link">Sponsor</a>
+                <a href="#" class="web-nav-link ">CN Campaign</a>
+            </div>
+        </div>
+    </header>
```

Comment:
- [ ] Kindly Your page makes a scroll bar when I resize less than `1250px`, kindly fix it to make your page responsive and to make your mobile version looks good. To fix it you can make their size smaller or you can make your header smaller to give you some space.

![image](https://user-images.githubusercontent.com/35535130/164779090-0222ecab-7b79-4097-9bbb-47ad1b701a08.png)

Reply:
Please make the necessary changes in your code so that this issue is solved 📝